datahub-web-react/src/app/entity/shared/containers/profile/useGetDataForProfile.ts

```diff
@@ -32,6 +32,7 @@ export default function useGetDataForProfile<T>({ urn, entityType, useEntityQuer
         refetch,
     } = useEntityQuery({
         variables: { urn },
+        fetchPolicy: 'cache-first',
```

Comment:
and this won't prevent showing updates when we add tags, terms, domains, documentation etc?

Reply:
testing locally things look alright to me... but never hurts to double check

Reply:
thats right -- generally we issue a refetch on changes and that will _always_ trigger a network call!